### PR TITLE
#4225. dialogs shift document

### PR DESCRIPTION
**Pull request by sshkalikov**

### CODE REVIEW

File: core/src/main/web/app/styles/modal.scss [hunk 1]
```diff
@@ -8,6 +8,10 @@ html {
     border-bottom-width: 0;
   }
 
+  .modal-open.has-scroll {
+    margin-right: 17px;
+  }
+
```
Comment (mnavasiolava) [inline core/src/main/web/app/styles/modal.scss:14]: hi, we have different scrollbar width for ambiance theme. so this style should also be overridden

**Comment by scottdraves** [comment]
that's much better but still shifts slightly for me.  check the line break of "Layout Managers":
screen shot 2016-06-08 at 10 34 58 am
screen shot 2016-06-08 at 10 34 48 am

**Comment by scottdraves** [comment]
please merge master to fix tests.

**Comment by scottdraves** [comment]
kick jenkins

**Pull request merged**